index.js

```diff
@@ -41,7 +41,6 @@ module.exports = function(source, map, meta) {
       if (typeof arg === "string") {
         arg = { bindings: arg };
       }
-      const forPath = !!arg.path;
```

Comment:
You're breaking `const fooPath = require("bindings")({ path: true, bindings: "foo.node", ... })` here.